src/xsar/sentinel1_meta.py

```diff
@@ -625,6 +625,12 @@ def image(self):
         img_dict['history'] = self.xml_parser.get_compound_var(self.files['annotation'].iloc[0], 'image', describe=True)
         return img_dict
 
+    @property
+    def azimuth_fmrate(self):
+        fmrates = self.xml_parser.get_compound_var(self.files['annotation'].iloc[0], 'azimuth_fmrate')
+        fmrates['history'] = self.xml_parser.get_compound_var(self.files['annotation'].iloc[0], 'azimuth_fmrate', describe=True)
```

Comment:
if `fmrates` is an xarray dataset, history should be in `fmrates.attrs['history']`, not in `fmrates['history']`

Reply:
Correction done!
![image](https://user-images.githubusercontent.com/14925067/157318194-833b3648-8b7c-4627-b98d-329538bad10a.png)